client/components/createTM/tab-setting.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import FormGroup from 'react-bootstrap/lib/FormGroup';
+import FormControl from 'react-bootstrap/lib/FormControl';
+import ControlLabel from 'react-bootstrap/lib/ControlLabel';
+import Col from 'react-bootstrap/lib/Col';
+
+export default class TabSetting extends Component {
```

Comment:
lets rename this file to `tab-setting-LOL.js`

Rename this component to `TabSettingLOL`